max7456.c

```diff
@@ -0,0 +1,733 @@
+/*
+ * Copyright (C) 2013 Chris
```

Comment:
this file should not have been added I guess

Reply:
If i understood you well...
The max7456.c file is needed as it has been changed,  a new osd nvram busy method is implemented that checks the osd nvram busy flag every time the osd display is written.

Reply:
I can see the changes in the file `sw/airborne/modules/display/max7456.c`, but their is also a `max7456.c` at the root directory. Are they the same or not ?

Reply:
What?
I don't have the slightest idea how this file got there, DELETED.
Only the files in the display directory are neede.